<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerWindow</class>
 <widget class="QMainWindow" name="ViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViewerWindow</string>
  </property>
  <widget class="M4D::GUI::Viewer::BasicSliceViewer" name="mViewer"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuViewer">
    <property name="title">
     <string>Viewer</string>
    </property>
    <addaction name="actionReset_View"/>
    <addaction name="separator"/>
    <addaction name="action2D"/>
    <addaction name="action3D"/>
    <addaction name="separator"/>
    <addaction name="actionEnable_Shading"/>
    <addaction name="actionEnable_Jittering"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuViewer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="viewerToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="action2D"/>
   <addaction name="action3D"/>
   <addaction name="separator"/>
   <addaction name="actionReset_View"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action2D">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2D</string>
   </property>
  </action>
  <action name="action3D">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3D</string>
   </property>
  </action>
  <action name="actionEnable_Shading">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Shading</string>
   </property>
  </action>
  <action name="actionEnable_Jittering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Jittering</string>
   </property>
  </action>
  <action name="actionReset_View">
   <property name="text">
    <string>Reset View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>M4D::GUI::Viewer::BasicSliceViewer</class>
   <extends>QWidget</extends>
   <header>GUI/widgets/BasicSliceViewer.h</header>
   <container>1</container>
   <slots>
    <slot>FineRender()</slot>
    <slot>EnableShading(bool)</slot>
    <slot>SaveCurrentView()</slot>
    <slot>ResetView()</slot>
    <slot>SaveCycle()</slot>
    <slot>EnableJittering(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>ViewerWindow</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEnable_Shading</sender>
   <signal>toggled(bool)</signal>
   <receiver>mViewer</receiver>
   <slot>EnableShading(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEnable_Jittering</sender>
   <signal>toggled(bool)</signal>
   <receiver>mViewer</receiver>
   <slot>EnableJittering(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReset_View</sender>
   <signal>triggered()</signal>
   <receiver>mViewer</receiver>
   <slot>ResetView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>applyTransferFunction()</slot>
  <slot>toggleInteractiveTransferFunction(bool)</slot>
  <slot>openFile()</slot>
 </slots>
</ui>
